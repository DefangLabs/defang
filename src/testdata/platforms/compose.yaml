services:
  intel2:
    platform: LINUX/X86_64
    image: busybox
