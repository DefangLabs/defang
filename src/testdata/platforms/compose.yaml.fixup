{
  "intel2": {
    "command": null,
    "deploy": {
      "replicas": 1,
      "resources": {},
      "placement": {}
    },
    "entrypoint": null,
    "image": "busybox",
    "networks": {
      "default": null
    },
    "platform": "LINUX/X86_64"
  }
}