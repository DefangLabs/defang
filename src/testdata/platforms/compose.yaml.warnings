 ! can only have more than one replica for PRO tier and PRODUCTION deployment mode. Will set replicas to 1 for service(s) [intel2]
 ! service "intel2": missing memory reservation; using provider-specific defaults. Specify deploy.resources.reservations.memory to avoid out-of-memory errors