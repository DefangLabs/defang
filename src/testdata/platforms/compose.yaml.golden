name: platforms
services:
  intel2:
    image: busybox
    networks:
      default: null
    platform: LINUX/X86_64
networks:
  default:
    name: platforms_default
